README.md

```diff
@@ -26,13 +26,153 @@ cd bidbot
+
+The SB collects all bids from connected miners and chooses one or multiple winners. The winning algorithm is still under development, but you can expect that the following facts will help winning an auction:
+
+- Provide a low price per epoch.
+- Provide a low DealStartEpoch.
+- Maintain a high deal success from past auctions.
+
+If you win an auction, you will be notified about it! If that weren't the case, you will see the declared winner published on the auction feed and most probably you'll have better luck in the next auction!
+
+The SB relies on offline deals to make deals with miners. After you win an auction, you will pull the deal data from the SB's IPFS nodes. You can think of this flow as an *automatic* offline deal setup where instead of receiving hard drives, you'll pull the data from someplace and then receive the offline-deal proposal.
+
+# How do I connect with the system?
+
+The hypothetical scenario in the previous section obviously didn't explain how you would really send bids or interact with the SB. The following diagram will help to get a better picture:
+
+![https://user-images.githubusercontent.com/6136245/124800912-97025680-df2c-11eb-9052-350adbad0d51.png](https://user-images.githubusercontent.com/6136245/124800912-97025680-df2c-11eb-9052-350adbad0d51.png)
+
+To connect to the system, you will install `bidbot`, a daemon that you run on your infrastructure. 
+
+The `bidbot` talks to the SB through libp2p pubsub topics. This is the same technology used by many `libp2p` applications such as `lotus`. Through these topics, your `bidbot` daemon will receive new *auctions* and can send *bids* to participate.
+
+Whenever you win an auction, your `lotus-miner` will receive the deal proposal for the corresponding data so you can accept it, and a *ProposalCid* is generated. After acceptance, your `bidbot` will automatically download the *PayloadCid* data from the SB's IPFS nodes, and import it as you generally do with offline deals, simulating the following commands you might be aware of: `lotus-miner storage-deals import-data <dealCid> <carFilePath>`
+
+The `carFilePath` is the CAR file that `bidbot` generated after downloading the deals data from the SB. The *dealCid* is the *ProposalCid* of the proposal you previously accepted from the SB.
+
+There's nothing more to do than the usual flow of deal execution steps from that moment forward. What's important is that the deal becomes active on-chain before the *StartDealEpoch* you promised in your bid!
+
+## How do I run `bidbot`?
+
+Here're the steps to do it:
+
+1. [Install Go 1.15 or newer](https://golang.org/doc/install)
+2. `git clone <https://github.com/textileio/broker-core.git>`
+3. `cd broker-core`
+4. `make install-bidbot` (check that `$GOPATH/bin` is part of your `$PATH`)
```

Comment:
Here should be changed. Maybe copy the `Getting Started` section.

Reply:
Agree, I'll copy the original get started here.